calibrators/marker_radar_lidar_calibrator/include/marker_radar_lidar_calibrator/marker_radar_lidar_calibrator.hpp

```diff
@@ -149,8 +164,9 @@ class ExtrinsicReflectorBasedCalibrator : public rclcpp::Node
 
   struct Parameters
   {
-    std::string radar_parallel_frame;  // frame that is assumed to be parallel to the radar (needed
-                                       // for radars that do not provide elevation)
+    std::string radar_optimization_frame;  // frame that is assumed to be parallel to the radar
```

Comment:
I think that this is not true anymore and as such the description should change.

For example, `base_link` will be used as the parallel frame when calibrating the ARS548 right?